nix/mobile/android/targets/release-android.nix

```diff
@@ -113,7 +113,7 @@ in stdenv.mkDerivation {
     ${concatStrings (catAttrs "shellHook" [ mavenAndNpmDeps status-go ])}
 
     pushd android
-    ${adhocEnvVars} gradle assemble${capitalizedBuildType} -S -Dmaven.repo.local='${localMavenRepo}' --offline ${gradle-opts} || exit
+    ${adhocEnvVars} ./gradlew -PversionCode=${build-number} assemble${capitalizedBuildType} || exit
```

Comment:
Wait, so when is it set to the actual value of the `versionNumber`, not the `9999`?

Reply:
Right, the command line arg overrides `gradle.properties`

Reply:
Right, but the `versionNumber` has to be set based on the result of `scripts/build_no.sh` in CI. How does that happen?

Reply:
That's happening here: https://github.com/status-im/status-react/blob/fix%2Frn-upgrade-misc/ci/utils.groovy#L69

Reply:
Ah yes, i was confused by the method in the Gradle file.